client/src/components/AllProjects/AllProjects.js

```diff
@@ -22,7 +23,7 @@ const AllProjects = props => (
                       (
                         <img
                           className="projectImage"
-                          src="https://raw.githubusercontent.com/Trie0037/Afterclass/master/client/public/favicon.png"
+                          src={defaults.defaultProjectImage}
```

Comment:
This default image URL was used three times in the app... very easy to break. I created a file that will store this URL so we can use it anywhere we want.